CHANGELOG.md

```diff
@@ -13,9 +13,17 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Extend cisco_interface_channel_group with attributes:
  - `channel_group_mode`
 
+- Added support for tftp and usb URIs to `cisco_upgrade`
+
+- Extend `cisco_upgrade` with attributes:
+ - `package`
+
 ### Changed
 
 ### Removed
+- Deprecated attributes for `cisco_upgrade`:
```

Comment:
I would also indicate that these properties are deprecated in favor of a single propertly `package` to manage the resource.